payment_paypal_multiwebsite_support/models/payment.py

```diff
@@ -16,7 +16,11 @@ class AcquirerPaypal(models.Model):
     def paypal_form_generate_values(self, values):
         paypal_tx_values = super(AcquirerPaypal,
                                  self).paypal_form_generate_values(values)
-        base_url = "https://" + request.httprequest.environ.get('HTTP_HOST', '')
+        base_url = ''
```

Comment:
Maybe no need to have this line (use `=` instead of `+=` in following lines)?

Reply:
Considering in case the "wsgi.url_scheme" might be empty

Reply:
Not sure how wsgi holds values, but shouldn't we explicitly return `http` in case url_scheme is empty?  Does your code still work when "wsgi.url_scheme" is empty?

Reply:
Yes, if "wsgi.url_scheme" is empty, the code will still work.
I believe it will be a rare case as it usually holds "http" or "https".

Reply:
> wsgi.url_scheme
> A string representing the “scheme” portion of the URL at which the application is being invoked. Normally, this will have the value “http” or “https”, as appropriate.
ref: http://wsgi.readthedocs.io/en/latest/definitions.html#envvar-wsgi.url_scheme